volumina/imageView2D.py

```diff
@@ -89,7 +89,7 @@ def hud(self, hud):
 
         scene.axesChanged.connect(hud.setAxes)
 
-    def __init__(self, parent, cropModel, imagescene2d):
+    def __init__(self, parent, cropModel, imagescene2d, editable):
```

Comment:
I'd prefer that the editable flag is set later, rather than changing the signature of a generic method only to pass that to a crop model